### PR TITLE
upgrade tools to 1.14

### PR DESCRIPTION
**Pull request by otan**
I think we can get rid of these as we have go.mod now, but /shrug my crlfmt is broken and I want it fixed sooner rather than later.

### CODE REVIEW

File: bin/update [hunk 1]
```diff
@@ -7,7 +7,7 @@ if [[ $# -ne 0 ]]; then
   exit 1
 fi
 
-go_version=go1.13
+go_version=go1.14
 xtools_version=86caa796c7ab0e32b908c5ebd5748faedd06c1da
```
Comment (otan) [inline bin/update:11]: where do we usually point xtools_version ...

**Comment by otan** [comment]
gofmt also changes very dramatically in 1.14 by the looks of it

File: bin/update (hunk 1 above)
Comment (bdarnell) [reply bin/update:11]: Whatever's current when we update the repo 🤷

**Review by bdarnell — APPROVED** [review]

**Comment by otan** [comment]
didn't end up needing this and i'd need to run go fmt across the whole codebase and that's yucky with older CRDB versions...
maybe someone else can take the reins :P

**Pull request closed without merging**